docs/swarm/admin_guide.md

```diff
@@ -0,0 +1,242 @@
+fault-tolerant. However, additional manager nodes reduce write performance
+because more nodes must acknowledge proposals to update the cluster state.
+This means more network round-trip traffic.
+
+Raft requires a majority of managers, also called a quorum to agree on proposed
+updates to the cluster. A quorum of managers must also agree on node additions
+and removals. Membership operations are subject to the same constraints as state
+replication.
+
+## Add manager nodes for fault tolerance
+
+You should maintain an odd number of managers in the swarm to support manager
+node failures. Having an odd number of managers ensures that during a network
+partition, there is a higher chance that a quorum remains available to process
+requests if the network is partitioned into two sets. Keeping a quorum is not
+guaranteed if you encounter more than two network partitions.
+
+| Cluster Size |  Majority  |  Fault Tolerance  |
+|:------------:|:----------:|:-----------------:|
+|      1       |     1      |         0         |
+|      2       |     2      |         0         |
+|    **3**     |     2      |       **1**       |
+|      4       |     3      |         2         |
+|    **5**     |     3      |       **2**       |
+|      6       |     4      |         2         |
+|    **7**     |     4      |       **3**       |
+|      8       |     5      |         3         |
+|    **9**     |     5      |       **4**       |
+
+
+For example, in a swarm with *5 nodes*, if you lose *3 nodes*, you don't have a
+quorum. Therefore you can't add or remove nodes until you recover one of the
+unavailable manager nodes or recover the cluster with disaster recovery
+commands. See [Recover from disaster](#recover-from-disaster).
+
+While it is possible to scale a swarm down to a single manager node, it is
+impossible to demote the last manager node. This ensures you maintain access to
+the swarm and that the swarm can still process requests. Scaling down to a
+single manager is an unsafe operation and is not recommended. If
+the last node leaves the cluster unexpetedly during the demote operation, the
+cluster swarm will become unavailable until you reboot the node or restart with
+`--force-new-cluster`.
+
+You manage cluster membership with the `docker swarm` and `docker node`
+subsystems. Refer to [Add nodes to a swarm](join-nodes.md) for more information
+on how to add worker nodes and promote a worker node to be a manager.
+
+## Distributing manager nodes
+
+In addition to maintaining an odd number of manager nodes, pay attention to
+datacenter topology when placing managers. For optimal fault-tolerance, distribute
+manager nodes across a minimum of 3 availability-zones or datacenter
+regions to support failures of an entire set of machines or common maintenance
+scenarios. If you suffer a failure in any of those zones, the swarm should maintain
+a quorum of manager nodes available to process requests and rebalance workloads.
```

Comment:
Also, since all control APIs and dispatcher APIs are proxied to the leader, the latency would be felt there too.

It would be interesting to actually test it with that kind of setup, and do store-intensive and dispatcher-intensive things like rolling updates.